### PR TITLE
Fixes-issue-7523-and-7657 with refactor of tutorials.md for 2.6.x

### LINKED ISSUE #7523: [doc] Tutorials page is out of date
The names of the tutorials and example projects are out of date:

https://www.playframework.com/documentation/2.6.x/Tutorials

Should be updated or refactored for 2.6.x

### PR DESCRIPTION
**Pull request by lousanna**
# Pull Request Checklist

* [x] Have you read [How to write the perfect pull request](https://github.com/blog/1943-how-to-write-the-perfect-pull-request)?
* [x] Have you read through the [contributor guidelines](https://www.playframework.com/contributing)?
* [x] Have you signed the [Lightbend CLA](https://www.lightbend.com/contribute/cla)?
* [x] Have you referenced any issues you're fixing using [commit message keywords](https://help.github.com/articles/closing-issues-using-keywords/)?
* [x] Have you added copyright headers to new files?
* [x] Have you checked that both Scala and Java APIs are updated?
* [x] Have you updated the documentation for both Scala and Java sections?
* [x] Have you added tests for any changed functionality?

## Fixes

Fixes #7523 and #7657.

## Purpose

-Changes links to 2.5.x documentation to 2.6.x.
-Moves the REST API project maintained by Play from the "Third Party" section to the "Play Maintained" section.
-Adds "Jump to" links at the top of Tutorials page.
-Adds links to video series under Third Party Tutorials.

### CODE REVIEW

File: documentation/manual/gettingStarted/Tutorials.md [hunk 1]
```diff
@@ -52,26 +56,32 @@ Play is non-opinionated about database access, and integrates with many object r
...
+* [playframework/play-anorm](https://github.com/playframework/play-anorm)
 
 #### JPA
 
 This is a example template showing Play with Java Persistence API (JPA), using Hibernate Entity Manager.  It is included in the Play project itself.
 
 * [play-java-intro](https://github.com/playframework/play-java-intro)
 
-#### Anorm
+#### EBean
```
Comment (richdougherty) [inline documentation/manual/gettingStarted/Tutorials.md]: I just noticed that the correct capitalisation is *Ebean* - see http://ebean-orm.github.io/. (I know the existing docs already had the wrong capitalisation!)

File: documentation/manual/gettingStarted/Tutorials.md [hunk 2]
```diff
@@ -119,6 +129,16 @@ This is an incomplete list of several helpful blog posts, and because some of th
 
 ### 2.5.x
 
+#### Play Framework Tutorial Video Series
```
Comment (richdougherty) [inline documentation/manual/gettingStarted/Tutorials.md]: Nice!

**Comment by lousanna** [comment]
Thanks @richdougherty! Applied your change.

**Comment by gmethvin** [comment]
2.6.x: 9488975

**Pull request merged**